src/ucp/rma/basic_rma.c

```diff
@@ -60,19 +76,38 @@ ucp_rma_request_advance(ucp_request_t *req, size_t frag_length,
     }
 }
 
+/**
```

Comment:
this comment should go inside the function:
function comment -> describe function API
internal code comment -> describe implementation